.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
- [ ] I would not recommend pushing unnecessary files or the file that have the important API key to make the app secure and also to look the project structure good. [Ex. remove `.env` file]